tests/sentry/api/endpoints/test_organization_historical_anomalies.py

```diff
@@ -0,0 +1,52 @@
+import orjson
+
+from sentry.incidents.models.alert_rule import (
+    AlertRuleSeasonality,
+    AlertRuleSensitivity,
+    AlertRuleThresholdType,
+)
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.helpers.features import with_feature
+from sentry.testutils.outbox import outbox_runner
+
+
+class OrganizationEventsAnomaliesEndpointTest(APITestCase):
```

Comment:
Can you add a test for the **less than 7 days historical data should return `[]`** case? Are there any additional error cases to cover?

Reply:
Going to add these once I implement the real Seer call!